examples/l3gd20/main.c

```diff
@@ -0,0 +1,36 @@
+/* vim: set sw=2 expandtab tw=80: */
+
+#include <l3gd20.h>
+#include <math.h>
+#include <stdio.h>
+#include <timer.h>
+
+#define ABS(N) ((N < 0) ? (-N) : (N))
+
+static int integer (float f)
```

Comment:
why have a function for this instead of just `(int)` cast?

Reply:
Thank you for the feedback.